src/base/variable.scss

```diff
@@ -60,6 +60,25 @@ $Whisper3: #ededed;
 $Gray: #8f8f8f;
 $PinkSwan: #b8b8b8;
 
+////////////////////////////////////JDS THEME COLOR////////////////////////////////////
+/// SUCCESS
+$JDSSuccessLightColor: #E9F7E9;
```

Comment:
Please check, colors are already available in existing variables.

Example: #E9F7E9 is $ColorFeedbackSuccess20

Reply:
Okay, I'll use an existing variable.